applications/dashboard/controllers/class.profilecontroller.php

```diff
@@ -541,7 +541,14 @@ public function invitations($userReference = '', $username = '', $userID = '') {
      * - 1: Unset the force cookie and use the user agent to determine the theme.
      */
     public function noMobile($type = 'desktop') {
-        if (!Gdn::request()->isAuthenticatedPostBack(true)) {
+        // Only validate CSRF for non guests.
```

Comment:
That's not true for open source. My hope here is that we can use the blame function of github and track down this PR.